Objective-C/CBLReplicatorConfiguration.h

```diff
@@ -35,6 +35,13 @@ typedef NS_ENUM(NSUInteger, CBLReplicatorType) {
     kCBLReplicatorTypePull                          ///< Pulling changes from the target
 };
 
+/** Network Interface experimental type. */
+typedef NS_ENUM(NSUInteger, CBLNetworkInterfaceExperimentalType) {
+    kCBLNetworkInterfaceExperimentalTypeUseGetAddrInfo = 0,     /// use the POSIX function getAddrInfo to specify the network interface.
```

Comment:
I would expect this option to be using DNSService to resolve the address instead of GetAddrInfo.